src/Sarif/Converters/FxCopConverter.cs

```diff
@@ -42,9 +42,12 @@ public void Convert(Stream input, IResultLogWriter output)
             }
 
             ToolInfo toolInfo = new ToolInfo();
-            RunInfo runInfo = new RunInfo();
             toolInfo.Name = "FxCop";
-            output.WriteToolAndRunInfo(toolInfo, runInfo);
+            output.WriteToolInfo(toolInfo);
+
+            // We can't infer/produce a runInfo object
```

Comment:
writing of toolInfo and runInfo are now broken into distinct steps. also, previously, writing the toolInfo initialized the log file (by writing root runLog and version details). as of your recently merged change, we do, in fact, produce a runInfo object in some circumstances (1 or more results generated)